tests/test_billing_context_rules.py

```
@@ -0,0 +1,77 @@
+        assignment="Neuron Installation",
+        task_text="Configured devices and staged inventory for go-live.",
+        work_date=date(2026, 5, 12),
+        start_time=time(9, 0),
+        end_time=time(17, 0),
+    )
+    assert context == "Configuration"
+    assert confidence == "high"
+
+
+def test_may_saturday_rule():
+    context, reason, confidence = resolve_work_context(
```

Ruff (0.15.14)

[warning] Unpacked variable `reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)